pkg/agent/params.go

```diff
@@ -227,9 +226,12 @@ func assignKubernetesParameters(properties *datamodel.Properties, parametersMap
 				// Kubernetes node binaries as packaged by upstream kubernetes
 				// example at https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md#node-binaries-1
 				addValue(parametersMap, "windowsKubeBinariesURL", kubernetesConfig.WindowsNodeBinariesURL)
+				addValue(parametersMap, "windowsContainerdURL", kubernetesConfig.WindowsContainerdURL)
```

Comment:
Missing
```

				// Kubernetes packages as zip file as created by Azure Pipelines
				// will be removed in future release as if gets phased out (https://github.com/Azure/aks-engine/issues/3851)
				kubeBinariesSASURL := kubernetesConfig.CustomWindowsPackageURL
				if kubeBinariesSASURL == "" {
					if properties.IsAzureStackCloud() {
						kubeBinariesSASURL = cloudSpecConfig.KubernetesSpecConfig.KubeBinariesSASURLBase + AzureStackPrefix + k8sComponents["windowszip"]
					} else {
						kubeBinariesSASURL = cloudSpecConfig.KubernetesSpecConfig.KubeBinariesSASURLBase + k8sComponents["windowszip"]
					}
				}
```

Reply:
Should we continue to support `IsAzureStackCloud` @yizhang4321  @AbelHu @xuto2

Reply:
The if/else is achieved in aks-rp, we can use `kubeBinariesSASURL` directly  here.